cordova-plugin-outline/apple/vpn/PacketTunnelProvider.m

```diff
@@ -272,8 +272,8 @@ - (void)connectTunnel:(NEPacketTunnelNetworkSettings *)settings
 
 - (NEPacketTunnelNetworkSettings *) getTunnelNetworkSettings {
   // TODO(alalama): check if the address is free, choose from pool.
-  NEIPv4Settings *ipv4Settings = [[NEIPv4Settings alloc] initWithAddresses:@[@"192.168.20.2"]
-                                                               subnetMasks:@[@"255.255.255.0"]];
+  NEIPv4Settings *ipv4Settings = [[NEIPv4Settings alloc] initWithAddresses:@[ @"192.168.20.1" ]
```

Comment:
I agree .1 is a...less surprising choice than .2 but can you remember why we chose .2 originally?

Reply:
Because, previously, the VPN's server address was at 192.168.20.1.